Detectors/CTP/Raw/include/CTPRaw/CTPRawReader.h

```diff
@@ -0,0 +1,28 @@
+#ifndef ALICEO2_CTP_CTPRAWREADER_H
+#define ALICEO2_CTP_CTPRAWREADER_H
+
+#include "DetectorsRaw/RawFileReader.h"
+
+namespace o2
+{
+namespace ctp
+{
+using namespace o2::raw;
+/**
+ * @brief The CTPRawReader class
+ * Code based on Raw/test/testRawReaderWriter
+ */
+class CTPRawReader
+{
+ public:
+  CTPRawReader() = default;
+  void init(const std::string& cfg);
+  int readRaw();
+
+ private:
+  std::unique_ptr<RawFileReader> mReader;
+  ClassDefNV(CTPRawReader, 1);
```

Comment:
@lietava  just noticed: you don't need RawFileReader, the raw data (i.e. binary chunk with set of RDH + payload for each link/FEEID) will come from upstream (if need to be loaded from file, will be done by https://github.com/AliceO2Group/AliceO2/tree/dev/Detectors/Raw#raw-data-file-reader-workflow.

What you need is a decoder device, which converts these raw data messages to CTP specific classes.